src/protocols/aeternity/libs/AeternityProtocol.ts

```diff
@@ -0,0 +1,30 @@
+import { Encoded } from '@aeternity/aepp-sdk';
+import { useStore } from 'vuex';
+import { useAeSdk } from '@/composables/aeSdk';
+import { BaseProtocol } from '@/protocols/BaseProtocol';
+import { PROTOCOL_AETERNITY } from '@/constants';
+
+type AeternityAddress = Encoded.AccountAddress;
+
+export class AeternityProtocol extends BaseProtocol {
+  static title = PROTOCOL_AETERNITY;
```

Comment:
Do we really need this?
In my opinion this class should be self enclosed, so it should not reach for any global stuff. Only types.